docs/tutorials/rootless_tutorial.md

```diff
@@ -163,21 +127,21 @@ runroot=""
 ```
 In rootless Podman these fields default to
 ```
-graphroot="$HOME/.local/share/containers/storage"
-runroot="$XDG_RUNTIME_DIR/containers"
+graphroot="${XDG_DATA_HOME}/containers/storage"
```

Comment:
also here XDG_DATA_HOME is not set by default so it should state the default used.